src/python/pants/engine/legacy/graph.py

```diff
@@ -417,6 +562,9 @@ def create_legacy_graph_tasks(symbol_table):
   """Create tasks to recursively parse the legacy graph."""
   symbol_table_constraint = symbol_table.constraint()
 
+  partial_find_owners = functools.partial(find_owners, symbol_table)
+  partial_find_owners.__name__ = find_owners.__name__
```

Comment:
Maybe extract a helper for this?

Reply:
Done already by stdlib: `functools.update_wrapper(functools.partial(find_owners, symbol_table), find_owners)`.